fmcapi/api_objects/policy_services/accessrules.py

```diff
@@ -82,7 +82,7 @@ class AccessRules(APIClassTemplate):
         "BLOCK_INTERACTIVE",
         "BLOCK_RESET_INTERACTIVE",
     ]
-    VALID_CHARACTERS_FOR_NAME = """[.\w\d_\- ]"""
+    VALID_CHARACTERS_FOR_NAME = """[.\w\d_\-\<\>\,\ ]"""
```

Comment:
Thanks for the commit @outlawww. The "<", ">", "," and " " characters are not special regex characters, so they don't need to be escaped with \. If you update that, I will test and release it tomorrow.

Reply:
oh yeah, you're right :) I'll change that.